src/pkg/crypto/eddsa.go

```diff
@@ -19,19 +18,3 @@ func NewBN254() (eddsa.PrivateKey, error) {
 	// Same hFunc should be used for sign and verify
 	return eddsa.GenerateKey(bytes.NewReader(seed))
 }
-
-func ImportBN256(importedPrivKey string) (eddsa.PrivateKey, error) {
```

Comment:
why did you remove this method as part of this PR?

Reply:
It's only used in one place `create_key` and it's mixing decoding and crypto operation in the same function.

Reply:
I removed the useless functions in `crypto/ecdsa` and `crypto/eddsa`